osmaxx/conversion/converters/utils.py

```diff
@@ -25,3 +34,11 @@ def zip_folders_relative(folder_list, zip_out_file_path=None):
         finally:
             os.chdir(old_dir)
     return zip_out_file_path
+
+
+def recursive_getsize(path):
+    size = 0
+    for entry in scandir(path):
+        if not entry.name.startswith('.') and entry.is_file():
+            size += os.path.getsize(entry.path)
+    return size
```

Comment:
I think this could be written as
```python
return sum(
    os.path.getsize(entry.path)
        for entry in scandir(path)
        if not entry.name.startswith('.') and entry.is_file()
)
```

Not sure which one is easier to grasp.